monorepo/monorepo-ws/apps/federation-gateway/src/dependencies.ts

```diff
@@ -0,0 +1,16 @@
+import { Configuration, LocalStorageManager, PrismaManager, Storage, SupergraphManager } from "@monorepo-ws/supergraph-manager"
+import { superGraphFile, superGraphFileName, supergraphLocation } from "./configuration/config"
+
+const getConfiguration = (): Configuration => {
```

Comment:
🎉 super hezky zej si to dal sem jen par vecicek: 

- prosim pojmenuj to spis - container.ts, dic.ts (dependency injection container) 
- v tomhle souboru by se mely uz drzet ty instance - to je ten hlavnvi point, ne jen fce ktere je vytvari, myslenka zatim je aby jsi mel jedno misto kde je prakticky aplikace slozena

Reply:
přejmenováno